docker/tds/docker-compose.yml

```diff
@@ -20,25 +35,26 @@ services:
     ports:
       - "32843:8080"
     env_file: tds-docker.env
-  exam_mysql:
-    image: mysql
-    ports:
-      - "32777:3306"
-    volumes_from:
-      - exam_mysql_data
-    env_file: tds-docker.env
-  exam_mysql_data:
-    image: mysql
-    entrypoint: /bin/bash
   exam:
     image: fwsbac/tds-exam-service
     ports:
       - "80:8080"
     depends_on:
-      - assessment
-      - session
-      - student
-      - config
-      - exam_mysql
+      configuration-service:
+        condition: service_healthy
+      assessment:
+        condition: service_started
+      session:
+        condition: service_started
+      student:
+        condition: service_started
+      config:
+        condition: service_started
     env_file:
-      - ./tds-docker.env
+      - tds-common.env
+      - exam.env
```

Comment:
I'm breaking the tds-docker.env into a tds-common.env for shared environment variables, with a per-service environment variable so that we can vary the profiles per-service. (For instance, we want the cache-enabled profile for the assessment and config services, but we want it disabled for the other services.)